### PR TITLE
[Security] Small optimization in AccessDecisionManager

### PR DESCRIPTION
**Pull request by jderusse**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | na |
| License | MIT |
| Doc PR | na |
- Remove unused variable `$abstain`
- Remove equals comparison: if not `$grant > $deny` and not `$deny > $grant` then `$grant` equals `$deny`

### CODE REVIEW

**Comment by WouterJ** [comment]
:+1: 

Status: Reviewed

**Comment by stof** [comment]
:+1:

File: src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php [hunk 1]
```diff
@@ -180,7 +174,7 @@ private function decideConsensus(TokenInterface $token, array $attributes, $obje
             return false;
         }
 
-        if ($grant == $deny && $grant != 0) {
+        if ($grant > 0) {
```
Comment (Nicofuma) [inline src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php]: This is wrong, you still need to check if there is the same number of grant and denial

**Comment by xabbuh** [comment]
:+1:

**Pull request merged**